config/initializers/download_processor.rb

```diff
@@ -0,0 +1,4 @@
+module Gemcutter
+  # Feature flag for counting downloads through stats-update or FastlyLogProcessor
+  ENABLE_FASTLY_LOG_PROCESSOR = Rails.env.test? || Rails.env.development?
```

Comment:
> why do we need this constant here? can we move this to the Download model or the log processor itself?

We don't need it *here*, but we need it somewhere. :smile:

Since it's serving as a feature flag, it's more appropriate as config than as app or library code. My rule of thumb is that app code can talk to config code; and config code can use `Rails.env`. But app code should never directly check `Rails.env`.

`app <--> config <--> environments`

FWIW, this is a temporary feature flag while we dark-launch Fastly log processing. Once we're confident it's working well, we'd remove this.